### PR TITLE
Improve account creation wizard layout

### PR DESCRIPTION
**Pull request by m4dz**
This PR improve account creation wizard layout and introduces some behaviors (like autofocus, or credential fields auto-fading)

### CODE REVIEW

File: client/app/components/accounts/server.cjsx [hunk 1]
```diff
@@ -118,3 +111,13 @@ module.exports = AccountServer = React.createClass
         _.each state, (value, key) ->
             globalState["#{protocol}#{key[0].toUpperCase()}#{key.slice 1}"] = value
         @props.onChange globalState
+
+
+    _securityOptions: ->
```
Comment (misstick) [inline client/app/components/accounts/server.cjsx]: Why are you using `_`as a prefix? I think that if you really want this method to be private, you should remove it from ReactComponent or do nothing else than removing the prefix :)

File: client/app/components/accounts/server.cjsx [hunk 2]
```diff
@@ -56,15 +56,8 @@ module.exports = AccountServer = React.createClass
...
-            .map (protocol) ->
-                value: protocol
-                label: t "server protocol #{protocol}"
-            .concat
-                value: 'none'
-                label: 'server protocol none'
-
-        <div className="content">
+        <div className="content server">
+            <h2>{@props._protocol}</h2>
```
Comment (misstick) [inline client/app/components/accounts/server.cjsx]: Why are you using `_`as a prefix? I think that if you really want this method to be private, you should remove it from ReactComponent or do nothing else than removing the prefix :)

**Comment by misstick** [comment]
👍 I'm waiting for other merge/rebase to merge this one

**Comment by m4dz** [comment]
@misstick ready to merge!

**Comment by m4dz** [comment]
@misstick rebase OK :+1:

**Pull request merged**